airbyte-integrations/connector-templates/source-configuration-based/source_{{snakeCase name}}/{{snakeCase name}}.yaml.hbs

```diff
@@ -1,6 +1,6 @@
 schema_loader:
   type: JsonSchema
-  file_path: "./source_{{snakeCase name}}/schemas/\{{ name }}.json"
+  file_path: "./source_{{snakeCase name}}/schemas/\{{ options.name }}.json"
```

Comment:
shouldn't this be `options.{{ name }}`?

Reply:
no, options is passed through for runtime interpolation.
https://github.com/airbytehq/airbyte/blob/master/airbyte-cdk/python/airbyte_cdk/sources/declarative/parsers/factory.py#L76-L95

unit tests: https://github.com/airbytehq/airbyte/blob/master/airbyte-cdk/python/unit_tests/sources/declarative/test_factory.py#L285

Reply:
I got confused between our templating and the template generator